lib/redmine.js

```diff
@@ -67,6 +67,7 @@ class Redmine {
       'issue_id': descriptionMatch[1],
       'spent_on': data.start.match(/^[^T]+/)[0],
       'hours': (data.duration * 1.2 / 3600).toFixed(1),
+      'comments': descriptionMatch.length === 3 ? descriptionMatch[2] : '',
```

Comment:
- You are assuming that this valid *only if there are 3 components*? What descriptionMatch gets extended?
- Why not: `descriptionMatch[2] || ''`?

Reply:
I was thinking about scenario when there is only one match. But that would be undefined so you are right on this.

Reply:
Better but still `#12345comment` will be matched, it is better to use `/#(\d+)(\s*:\s*(.+))?/`, you'd have to change group index from `2` to `3`. The latter regex matches:

- #12345 : comment
- #12345:comment
- #12345    :     comment
- etc.